mw-webapp/src/utils/auth/handleLogIn.ts

```diff
@@ -0,0 +1,10 @@
+import {signInWithRedirect} from "firebase/auth";
+import {auth, provider} from "src/firebase";
+
+export const handleLogIn = async () => {
+  try {
+    await signInWithRedirect(auth, provider);
```

Comment:
should it be in service layer? what heppend for user sign in throw error? Nothing except alert? Is it a good scenario?

Reply:
Didn't change the behavior when Logout error appears because I didn't understand what else needs to be done? Redirect to the authorization page?